config/rbac/manager_role.yaml

```diff
@@ -0,0 +1,188 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: controller-manager
+  namespace: kcp-system
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
Do we really need all verbs for config maps?

Reply:
Probably not. I reduced it to "read" policy rules.